src/browser/api/window.js

```diff
@@ -506,16 +499,6 @@ Window.create = function(id, opts) {
             _externalWindowEventAdapter = new ExternalWindowEventAdapter(browserWindow);
         }
 
-        let teardownListeners = () => {
-            // tear down any listeners...
-            _openListeners.forEach(unhook => {
-                unhook();
-            });
-
-            //tear down any listeners on external event emitters.
-            webContents.removeListener(OF_WINDOW_UNLOADED, ofUnloadedHandler);
```

Comment:
Removed this here, because another [PR](https://github.com/HadoukenIO/core/pull/413) is taking care of it